### DISCUSSION TITLE
Query syntax for using workflowRun and workflow object of GitHub GraphQL API?

### DISCUSSION THREAD

**Discussion by SaurabhRai19** [community/community#15203] [Other Feature Feedback, Questions, & Ideas]
Hi. I am facing an issue with writing the query that can display info of the workflows running inside a repository. There exists [workflowRun and workflow](https://docs.github.com/en/enterprise-server@3.3/graphql/reference/objects#workflow) object for this requirement. But can’t find the right syntax to use these objects for returning the workflow-related info.

**Comment by nesk** [comment]
This is because these 2 objects are only linked to the [CheckSuite](https://docs.github.com/en/graphql/reference/objects#checksuite) object and are not manually queryable, unfortunately.
